### PR TITLE
Implement new spacing values

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
## Description
New feature applied to `core/heading`.

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -88,11 +88,21 @@
...
+        "spacing": {
+          "margin": true
+        }
+      },
       "core/group": {
         "spacing": {
           "padding": true
         }
       },
+      "p4/issues": {
```
Comment (GP-Dan-Tovbein) [inline theme.json:101]: This is needed by https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/914

File: theme.json [hunk 2]
```diff
@@ -88,11 +88,21 @@
           "radius": false
         }
       },
+      "core/heading": {
```
Comment (GP-Dan-Tovbein) [inline theme.json:91]: Also needed by https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/914 but of course by the rest of blocks / patterns.

Comment (mleray) [reply theme.json:91]: I don't see how it's needed in the PR that you referenced here 🤔 And I'm not sure we have other patterns that would require this!

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [deimos](https://www-dev.greenpeace.org/test-deimos) | [admin](https://www-dev.greenpeace.org/test-deimos/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-deimos/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-deimos) | [composer-local.json](https://github.com/greenpeace/planet4-test-deimos/blob/main/composer-local.json)

:watch: 2022.08.04 15:22:42

File: theme.json (hunk 2 above)
Comment (GP-Dan-Tovbein) [reply theme.json:91]: Removed that and keep only the feature to `core/heading`

**Review by mleray — APPROVED** [review]
Tested on local, looks good to me 👍

**Pull request merged**